### PR TITLE
Changed default DID mode

### PR DESCRIPTION
**Pull request by Maneesha-rubix**
Setting Basic mode as the default mode and Light mode as fourth mode while creating DID.

### CODE REVIEW

File: did/model.go [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package did
 
 const (
-	LightDIDMode int = iota
-	BasicDIDMode
+	BasicDIDMode int = iota
 	StandardDIDMode
 	WalletDIDMode
+	LightDIDMode
```
Comment (harirubix) [inline did/model.go:7]: Could have added as option 4 rather than 3 . It works fine now since we dont usually make wallets with child. But this might open some corner case issues during switch or if/else statements.

**Review by harirubix — APPROVED** [review]
Code looks fine.  Please test different scenarios before merging

File: did/model.go (hunk 1 above)
Comment (Maneesha-rubix) [reply did/model.go:7]: In that case I will change the sequence again.

**Review by vaishnav-ch — APPROVED** [review]

**Review by harirubix — APPROVED** [review]
